doc/changes/latest.inc

```diff
@@ -92,6 +92,8 @@ Bugs
 
 - Fix bug in the :class:`mne.viz.Brain` tool bar that prevented the buttons to call the corresponding feature (:gh:`10560` by `Guillaume Favelier`_)
 
+- Fix issue with saving epochs once :func:`mne.preprocessing.compute_current_source_density` has been used if a rejection threshold was used first (:gh:`10619` by `Alex Rockhill`_ and `Richard Höchenberger`_)
```

Comment:
```suggestion
- Fix issue with saving epochs once :func:`~mne.preprocessing.compute_current_source_density` has been used if a rejection threshold was used first (:gh:`10619` by `Alex Rockhill`_ and `Richard Höchenberger`_)
```

Reply:
What does this do?

Reply:
only prints "compute_current_source_density" while still linking to "mne.preprocessing.compute_current_source_density"

Reply:
Nice